R/optparse.R

```diff
@@ -171,6 +173,7 @@ OptionParser <- function(usage = "usage: %prog [options]", option_list=list(),
 #'     value if the option is found on the command string.  \dQuote{store_true}
 #'     stores \code{TRUE} if the option is found and \dQuote{store_false} stores
 #'     \code{FALSE} if the option is found.
+#' @param callback A function that executes after the each option value is fully parsed
```

Comment:
New parameters``callback_args`` and ``callback_kwargs`` seem to be undocumented.  This should have given you a warning when you ran ``devtools::check()``.  ``callback`` documentation could also be made more clear:  

    @param callback A function that executes after the each option value is fully parsed and whose value is assigned to the option.  All callbacks are called as \code{callback(option_object, long_flag, value, parser_object, callback_args, callback_kwargs)}.